release/smbchk.py

```diff
@@ -45,6 +45,13 @@
     'cyclus::Composition::NewDecay',
 }
 
+CPP11_SYMBOLS_REPLACEMENTS = [
```

Comment:
Maybe this name should be changed to `SYMBOL_REPLACEMENTS` if it is extended beyond C++11